js/apps/system/_admin/aardvark/APP/frontend/js/collections/arangoCollections.js

```diff
@@ -167,7 +167,11 @@
         data.shardKeys = object.shardBy;
       }
       if (object.replicationFactor) {
-        data.replicationFactor = JSON.parse(object.replicationFactor);
+        if (typeof object.replicationFactor === 'number' && frontendConfig.isEnterprise) {
+          data.replicationFactor = JSON.parse(object.replicationFactor);
+        } else {
+          data.replicationFactor = object.replicationFactor;
+        }
       }
```

Comment:
@hkernbach Can you fix and update the branch `feature/ui-satellites` (just push to it)? This will also update the PR.